sdk/tests/conformance2/extensions/ext-color-buffer-float.html

```diff
@@ -0,0 +1,312 @@
+    }
+}
+</script>
+<!-- Shaders for testing floating-point render targets -->
+<script id="positionVertexShader" type="x-shader/x-vertex">
+attribute vec4 vPosition;
+void main()
+{
+    gl_Position = vPosition;
+}
+</script>
+<script id="floatingPointFragmentShader" type="x-shader/x-fragment">
+void main()
+{
+    gl_FragColor = vec4(10000.0, 10000.0, 10000.0, 10000.0);
+}
+</script>
+<script>
+"use strict";
+description("This test verifies the functionality of the EXT_color_buffer_float extension, if it is available.");
+
+debug("");
+
+var wtu = WebGLTestUtils;
+var canvas = document.getElementById("canvas");
+var gl = wtu.create3DContext(canvas, null, 2);
+
+if (!gl) {
+  testFailed("WebGL context does not exist");
+} else {
+  testPassed("WebGL context exists");
+
+  var texturedShaders = [
+      wtu.setupSimpleTextureVertexShader(gl),
+      "testFragmentShader"
+  ];
+  var testProgram =
+      wtu.setupProgram(gl,
+                       texturedShaders,
+                       ['vPosition', 'texCoord0'],
+                       [0, 1]);
+  var quadParameters = wtu.setupUnitQuad(gl, 0, 1);
+
+  // Ensure these formats can't be used for rendering if the extension is disabled
+  runFloatTextureRenderTargetTest(false, testProgram, gl.R16F, gl.RED);
+  runFloatTextureRenderTargetTest(false, testProgram, gl.RG16F, gl.RG);
+  runFloatTextureRenderTargetTest(false, testProgram, gl.R32F, gl.RED);
+  runFloatTextureRenderTargetTest(false, testProgram, gl.RG32F, gl.RG);
+  runFloatTextureRenderTargetTest(false, testProgram, gl.RGBA32F, gl.RGBA);
+  runFloatTextureRenderTargetTest(false, testProgram, gl.R11F_G11F_B10F, gl.RGB);
+
+  runFloatRenderbufferRenderTargetTest(false, testProgram, gl.R16F);
+  runFloatRenderbufferRenderTargetTest(false, testProgram, gl.RG16F);
+  runFloatRenderbufferRenderTargetTest(false, testProgram, gl.R32F);
+  runFloatRenderbufferRenderTargetTest(false, testProgram, gl.RG32F);
+  runFloatRenderbufferRenderTargetTest(false, testProgram, gl.RGBA32F);
+  runFloatRenderbufferRenderTargetTest(false, testProgram, gl.R11F_G11F_B10F);
+
+  if (!gl.getExtension("EXT_color_buffer_float")) {
+      testPassed("No EXT_color_buffer_float support -- this is legal");
+  } else {
+      testPassed("Successfully enabled EXT_color_buffer_float extension");
+      runFloatTextureRenderTargetTest(true, testProgram, gl.R16F, gl.RED, 1, [10000, 1, 1, 1], 0);
+      runFloatTextureRenderTargetTest(true, testProgram, gl.RG16F, gl.RG, 2, [10000, 10000, 1, 1], 0);
+      runFloatTextureRenderTargetTest(true, testProgram, gl.R32F, gl.RED, 1, [10000, 1, 1, 1], 0);
+      runFloatTextureRenderTargetTest(true, testProgram, gl.RG32F, gl.RG, 2, [10000, 10000, 1, 1], 0);
+      runFloatTextureRenderTargetTest(true, testProgram, gl.RGBA32F, gl.RGBA, 4, [10000, 10000, 10000, 10000], 0);
+      runFloatTextureRenderTargetTest(true, testProgram, gl.R11F_G11F_B10F, gl.RGB, 3, [9984, 9984, 9984, 1], 0); // Different values to account for lower precision
```

Comment:
We could write out 1,000 to the shader instead of 10,000, which would eliminate the precision issue here. I only used 10,000 because that's what the OES_texture_float test used.